requirements/requirements_dev.in

```diff
@@ -1,10 +1,12 @@
--r requirements.txt
--c requirements.txt
+-r requirements.in
+-c requirements.in
 pytest>=6.2.5
 pytest-cov>=3.0.0
-pip-tools>=6.4.0
-pre-commit>=2.16.0
+pip-tools==6.1.0
```

Comment:
What is wrong with newer versions of pip-tools? Why limit this template to a specific (closed) version?

Reply:
Nothing, but I was just keeping this consistent with the version present in .pre-config-config.yaml. If we are not doing this, the two versions of pip-tools produce a slightly different requirements.txt and requirements_dev.txt files, and therefore this make pre-commit checks to NOT pass. If one wants to use pre-commit, the version of pip-tools MUST be the same between the environment and the pre-commit checks